README.md

```diff
@@ -35,22 +35,46 @@ A GitHub app with the following permissions is required:
 - `Read access to metadata`
 - `Read and write access to administration, code, and issues`
 
-### Variables
+To be able to perform git operations using the apps identity retrieved the apps username and email as follows:
+
+It is possible to perform git operations using the apps identity. Retrieve the required username and email as follows:
```

Comment:
```suggestion
It is possible to perform git and GitHub operations using the apps identity. Retrieve the required username and email as follows:
```

Reply:
only used for git

Reply:
Removed the GitHub part from the suggestion @m4s-b3n